static/app/components/codeSnippet.tsx

```diff
@@ -80,6 +82,11 @@ export function CodeSnippet({
 }: CodeSnippetProps) {
   const ref = useRef<HTMLModElement | null>(null);
 
+  // https://prismjs.com/plugins/line-highlight/
+  useEffect(() => {
+    import('prismjs/plugins/line-highlight/prism-line-highlight');
```

Comment:
You could add a `if (linesToHeightlight)` condition to avoid importing if we aren't using the feature.

Reply:
Done in next commit